playbooks/edx-east/edxapp.yml

```diff
@@ -9,7 +9,6 @@
   roles:
     - aws
     - role: scorm
-      when: "{{ EDXAPP_XBLOCK_SETTINGS }}.get('ScormXBlock', False)"
```

Comment:
Gotcha, the role itself will decide if it will run its internals because of the ansible < 2.0 issue you mention